react/ui-components/src/hoc/EditablePopup.js

```
@@ -0,0 +1,130 @@
+      ]}
+      footerChildren={[
+        <Button
+          type={'button'}
+          size={'large'}
+          variation={'primary'}
+          label={t('YES')}
+          title={t('YES')}
+          onClick={() => {
+            console.log('action clicked');
+          }}
+        />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)